.github/workflows/build_impl.yml

```diff
@@ -115,8 +115,7 @@ jobs:
         run: |
           ClangBuildAnalyzer --all build/ build_time_report.bin
           ClangBuildAnalyzer --analyze build_time_report.bin > build_time_report.txt
-          # Running again to have colorized output
-          ClangBuildAnalyzer --analyze build_time_report.bin
```

Comment:
I checked and this tool doesn't produce colorful output in GitHub Actions, so it's better not to run it twice.